### ISSUE TITLE
Resource channel; suggestion from WiLD

### ISSUE THREAD

**Issue by jdotrjs** [phaser-discord/administration#33]
Suggested by WiLD#4832 / 180708488144486400, on Sunday, April 29, 2018, 19:54 UTC:
>a resource channel would be nice

**Comment by AleBles** [comment]
We could use github wiki for that, would also be a solid place to add resources / links and what not

**Comment by jdotrjs** [comment]
Agreed. My rough thought was to keep throwing things into a FAQ / various markdown docs until they became too hard to navigate or so many folks wanted to contribute I couldn't keep up with PRs (that hasn't been the case :wink:) and then fall back to a wiki.

Eventually I'd like for us to be managing a github pages space with a bunch of docs but that's far enough out that I haven't really put much thought into it.

**Comment by 16patsle** [comment]
Some recent discussion on this from #discord-feedback:

Any Key#8000
> We have a resources channel in a guild I’m part of that’s not engine specific and it’s basically a wall of links at this point. If you know that a resource was posted you can search the channel, the single channel not being granular makes it a bit of an issue. If you want resource channels that are a good reference I would suggest a group if channels with specific ones dedicated to some group/type of resources.

TheEBBGaming#5732
> A wiki would be extremely helpful
sometimes the plain documentation just confuses me more

nfl_matt#0687
> I would be very appreciative of a well curated wiki.

Elfayer#8549
> Too bad there is no channel to share links, like in many discord dev servers. Something like #awesome-links #news-and-links :thinking:
+1 for the wiki. I'd love something along the line of discord bots community guide: https://discordjs.guide/ . It's made using https://vuepress.vuejs.org/
